src/Entity/Storage/MonetizationEntityStorageBase.php

```diff
@@ -0,0 +1,337 @@
+<?php
+/**
+ * Copyright 2018 Google Inc.
+ *
+ * This program is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU General Public License version 2 as published by the
+ * Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY
+ * or FITNESS FOR A PARTICULAR PURPOSE. See the GNU General Public
+ * License for more details.
+ *
+ * You should have received a copy of the GNU General Public License along
+ * with this program; if not, write to the Free Software Foundation, Inc., 51
+ * Franklin Street, Fifth Floor, Boston, MA 02110-1301, USA.
+ */
+
+namespace Drupal\apigee_m10n\Entity\Storage;
+
+use Apigee\Edge\Api\Monetization\Entity\EntityInterface as MonetizationEntityInterface;
+use Drupal\Component\Datetime\TimeInterface;
+use Drupal\Core\Cache\Cache;
+use Drupal\Core\Cache\CacheBackendInterface;
+use Drupal\Core\Entity\EntityInterface;
+use Drupal\Core\Entity\EntityStorageBase;
+use Drupal\Core\Entity\EntityTypeInterface;
+use Psr\Log\LoggerInterface;
+use Symfony\Component\DependencyInjection\ContainerInterface;
+
+abstract class MonetizationEntityStorageBase extends EntityStorageBase {
```

Comment:
This is also a copy-paste clone :skull_and_crossbones:  of https://github.com/apigee/apigee-edge-drupal/blob/8.x-1.x/src/Entity/Storage/EdgeEntityStorageBase.php. Why did you extended the EdgeEntityStorageBase class instead?